src/pages/dar_application/ResearcherInfo.js

```diff
@@ -228,7 +228,7 @@ export default function ResearcherInfo(props) {
               }),
               span({
                 className: 'cancel-color required-field-error-span', isRendered: (invalidInvestigator) && (showValidationMessages)
-              }, ['Required field'])
+              }, ['Required field, your Profile must have a PI'])
```

Comment:
This, I think, is not technically true. A researcher can also BE the PI and in that case, not need to have a separate PI listed. In testing, it looks like this validation isn't working. I can turn off my PI status and have not PI listed:

----
<img width="374" alt="Screen Shot 2021-03-26 at 9 40 06 AM" src="https://user-images.githubusercontent.com/116679/112640004-4e101180-8e17-11eb-95b1-838e2c1f2b4e.png">

----
HOWEVER, I can still submit a DAR this way, so I don't think this message shows up. The PI field isn't selectable based on the user profile:

----
<img width="1004" alt="Screen Shot 2021-03-26 at 9 43 29 AM" src="https://user-images.githubusercontent.com/116679/112640605-dc849300-8e17-11eb-958e-e229ec5b8250.png">

----
I think this should be invalid from the start if the user's profile isn't the PI and doesn't have a PI. We need some form of messaging on the DAR form that explains this when the PI field isn't selectable like it is in the screenshot.

Reply:
Good catch! I updated the comment to be more clear and removed the '--' placeholder that allows the DAR to be submitted when it shouldn't. I also updated the ticket to reflect this